src/Mvc/Mvc.Core/src/Formatters/SystemTextJsonOutputFormatter.cs

```diff
@@ -33,6 +33,19 @@ public SystemTextJsonOutputFormatter(JsonOptions options)
             SupportedMediaTypes.Add(MediaTypeHeaderValues.ApplicationAnyJsonSyntax);
         }
 
+        internal static SystemTextJsonOutputFormatter CreateFormatter(JsonOptions jsonOptions)
+        {
+            var jsonSerializerOptions = jsonOptions.JsonSerializerOptions;
+
+            if (jsonSerializerOptions.Encoder is null)
```

Comment:
🆙 📅 

This limits the encoder to use the relaxed encoding specifically in the output formatter assuming the user hasn't already configured it.

Reply:
Spoke to @GrabYourPitchforks offline. We decided limited this to the formatter is reasonable. The JsonHelper always defaults to using the Default encoding.